docs/issues/58ee357bfd50147615170c189a8a14c6.md

```
@@ -1,30 +1,49 @@
-## Follow-up
+Main thread ANR occurring during the handling of an uncaught exception by the Firebase Crashlytics SDK.
+
+## Details
+- **Issue ID:** 58ee357bfd50147615170c189a8a14c6
+- **Error Type:** ANR
+- **First Seen:** 2.8.0
+- **Last Seen:** 2.8.0
+- **State:** OPEN
+
+## Stack Trace
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)